tests/python/rest_api/test_tasks.py

```diff
@@ -809,16 +809,17 @@ def test_can_create_task_with_exif_rotated_images(self):
         task_id, _ = create_task(self._USERNAME, task_spec, task_data)
 
         # check that the frames have correct width and height
-        with make_api_client(self._USERNAME) as api_client:
-            _, response = api_client.tasks_api.retrieve_data(
-                task_id, number=0, type="chunk", quality="original"
-            )
-            with zipfile.ZipFile(io.BytesIO(response.data)) as zip_file:
-                for name in zip_file.namelist():
-                    with zip_file.open(name) as zipped_img:
-                        im = Image.open(zipped_img)
-                        # original is 480x640 with 90/-90 degrees rotation
-                        assert im.height == 640 and im.width == 480
+        for chunk_quality in ["original", "compressed"]:
+            with make_api_client(self._USERNAME) as api_client:
+                _, response = api_client.tasks_api.retrieve_data(
+                    task_id, number=0, type="chunk", quality=chunk_quality
+                )
+                with zipfile.ZipFile(io.BytesIO(response.data)) as zip_file:
+                    for name in zip_file.namelist():
+                        with zip_file.open(name) as zipped_img:
+                            im = Image.open(zipped_img)
+                            # original is 480x640 with 90/-90 degrees rotation
+                            assert im.height == 640 and im.width == 480
```

Comment:
I think it would be nice to check meta as well (example [here](https://github.com/opencv/cvat/blob/bb579e83cae6fac36cd4b65e29bade93b21be412/tests/python/rest_api/test_tasks.py#L939-L944)).